File: app/src/main/java/com/example/a18/path/bezier/BezierActivity.java
package com.example.a18.path.bezier;

import android.app.Activity;
import android.os.Bundle;
import android.view.WindowManager;

import com.example.a18.path.R;

public class BezierActivity extends Activity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        getWindow().setFlags(WindowManager.LayoutParams.FLAG_FULLSCREEN,WindowManager.LayoutParams.FLAG_FULLSCREEN);

        setContentView(R.layout.activity_bezier2);

    }
}

File: bezier/src/main/java/pdog18/com/module_bezier/BezierActivity.kt
package pdog18.com.module_bezier

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import android.view.WindowManager
import com.alibaba.android.arouter.facade.annotation.Route

@Route(path = "/module/bezier")
class BezierActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        window.setFlags(WindowManager.LayoutParams.FLAG_FULLSCREEN, WindowManager.LayoutParams.FLAG_FULLSCREEN)

        setContentView(R.layout.activity_main)
    }
}
